legal-api/src/legal_api/models/db.py

```diff
@@ -270,13 +131,8 @@ def after_transaction_create(session, transaction):
     def clear_transaction(session, transaction):
         VersioningProxy.unlock_versioning(session, transaction)
 
-    _new_enable_versioning(transaction_cls=Transaction)
-    make_versioned(user_cls=None, manager=versioning_manager)
+    enable_versioning(transaction_cls=Transaction)
```

Comment:
`VersioningProxy` and `setup_versioning()` were introduced to support switching between the old and the new versioning approaches. They can be removed in the end. `enable_versioning(transaction_cls=Transaction)` can be called directly during initialization (e.g. line 138).